### PR TITLE
Add missing periods (System.Windows)

### PR DESCRIPTION
**Pull request by gewarren**

### CODE REVIEW

File: xml/System.Windows.Forms/StatusBar.xml [hunk 1]
```diff
@@ -1379,7 +1379,7 @@ This class is not available in .NET 5.0 and later versions. Use the <see cref="T
       <Parameters />
       <Docs>
         <summary>Returns a string representation for this control.</summary>
-        <returns>String</returns>
+        <returns>To be added.</returns>
```
Comment (mairaw) [inline xml/System.Windows.Forms/StatusBar.xml:1382]: It does return something: https://referencesource.microsoft.com/#System.Windows.Forms/winforms/Managed/System/WinForms/StatusBar.cs,50566e4e57e7fc20

Better to still be compliant

```suggestion
        <returns>A string representation for this control.</returns>
```

File: xml/System.Windows.Ink/DrawingAttributes.xml [hunk 2]
```diff
@@ -874,7 +874,7 @@
         <Parameter Name="e" Type="System.ComponentModel.PropertyChangedEventArgs" />
       </Parameters>
       <Docs>
-        <param name="e">EventArgs</param>
+        <param name="e">To be added.</param>
```
Comment (mairaw) [inline xml/System.Windows.Ink/DrawingAttributes.xml:877]: https://github.com/dotnet/dotnet-api-docs/wiki/Parameters:-OnEvent---Methods

```suggestion
        <param name="e">The event data.</param>
```

File: xml/System.Windows.Interop/HwndSourceParameters.xml [hunk 3]
```diff
@@ -1001,7 +1001,7 @@
       </ReturnValue>
       <Docs>
         <summary>Gets or sets the style for the window.</summary>
-        <value>The window style. See the [CreateWindowEx](https://go.microsoft.com/fwlink/?LinkId=98462) function for a complete list of style bits. Defaults: WS_VISIBLE, WS_CAPTION, WS_SYSMENU, WS_THICKFRAME, WS_MINIMIZEBOX, WS_MAXIMIZEBOX, WS_CLIPCHILDREN</value>
+        <value>The window style. See the [CreateWindowEx](https://go.microsoft.com/fwlink/?LinkId=98462) function for a complete list of style bits. Defaults: WS_VISIBLE, WS_CAPTION, WS_SYSMENU, WS_THICKFRAME, WS_MINIMIZEBOX, WS_MAXIMIZEBOX, WS_CLIPCHILDREN.</value>
```
Comment (mairaw) [inline xml/System.Windows.Interop/HwndSourceParameters.xml:1004]: Better to use the docs link?

```suggestion
        <value>The window style. See the [CreateWindowEx](/windows/win32/api/winuser/nf-winuser-createwindowexa) function for a complete list of style bits. Defaults: WS_VISIBLE, WS_CAPTION, WS_SYSMENU, WS_THICKFRAME, WS_MINIMIZEBOX, WS_MAXIMIZEBOX, WS_CLIPCHILDREN.</value>
```

**Review by mairaw — APPROVED** [review]
LGTM. Left a couple of comments for you to consider.

File: xml/System.Windows.Forms/StatusBar.xml (hunk 1 above)
Comment (gewarren) [reply xml/System.Windows.Forms/StatusBar.xml]: Thanks for catching these. I thought I'd double-checked all my diffs for places where I'd replaced with "To be added".

**Pull request merged**